src/components/name/create-name.js

```diff
@@ -1,92 +1,125 @@
-                    </div>
-                    <div className="form-group">
-                        <label htmlFor="origin_language">Origin Language</label>
-                        <select value={origin_language}
-                            onChange={e => setOriginLanguage(e.target.value)} className="form-control" id="origin_language">
-                                <option>Select Origin Language</option>
-                            {
-                                allOrigins.map( (origin) => {
-                                return (<option key={origin.origin_id} value={origin.origin_id}>{origin.title}</option>)
-                                })
-                            }
-                        </select>
-                    </div>
-                    <div className="form-group">
-                        <label htmlFor="gender">Gender</label>
-                        <select
-                            onChange={e => setGender(e.target.value)} className="form-control">
-                                <option value="">Select For which gender this name is for</option>
-                            {allGenders.map((gender) => {
-                                return (<option key={gender.gender_id} value={gender.gender_id}>{gender.title}</option>);
-                            })}
-                        </select>
-                    </div>
+  return (
+    <Fragment>
+      <div className="container">
+        <h3 className="text-center mt-100">{props.title}</h3>
+        <form onSubmit={onSubmitForm}>
+          <div className="form-group">
+            <label htmlFor="id">Name</label>
+            <input
+              type="text"
+              value={name}
+              onChange={(e) => {
+                setName(e.target.value);
+              }}
+              className="form-control"
+              id="name"
+              placeholder="Name"
+            />
+          </div>
+          <div className="form-group">
+            <label htmlFor="origin_language">Origin Language</label>
+            <select
+              value={origin_language}
+              onChange={(e) => setOriginLanguage(e.target.value)}
+              className="form-control"
+              id="origin_language"
+            >
+              <option>Select Origin Language</option>
+              {allOrigins.map((origin) => {
+                return (
+                  <option key={origin.origin_id} value={origin.origin_id}>
+                    {origin.title}
+                  </option>
+                );
+              })}
+            </select>
+          </div>
+          <div className="form-group">
+            <label htmlFor="gender">Gender</label>
+            <select
+              onChange={(e) => setGender(e.target.value)}
+              className="form-control"
+            >
+              <option value="">Select Applicable Gender</option>
```

Comment:
It looks like this is the only change we need to work on. We don't need to change all the other stuff. We will stick to change only what is required. As you can see here are tones of changes on this file and we don't want to have that.